docs/concepts.md

```diff
@@ -1,7 +1,7 @@
 # Oak Concepts
 
 - [Oak Runtime](#oak-runtime)
-- [Oak Node](#oak-node)
+- [Oak Node](#nodes)
```

Comment:
Could you make the text in-sync too (`[Nodes](#nodes)`) please?

Reply:
Done.